prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
what is this file used for?

Reply:
that is a reminder to me that I didn't git ignore that yet!